build.sbt

```diff
@@ -41,3 +43,20 @@ site.jekyllSupport()
 ghpages.settings
 
 git.remoteRepo := "git@github.com:willb/silex.git"
+
+lazy val silex = project in file(".")
+
+lazy val spark = project.dependsOn(silex)
+  .settings(commonSettings:_*)
```

Comment:
I think technically we need to have the Spark dependencies for the `spark` project _without_ `provided`, since there isn't anywhere else for them to come from.  (In a typical Silex app, the user would be supplying Spark dependencies elsewhere.)  This may not matter in practice because `console` runs under the `compile` scope IIRC, but it's maybe something to keep in mind.